packaging/RPMS/Fedora/rabbitmq-server.service

```diff
@@ -6,6 +6,7 @@ After=syslog.target network.target
 Type=notify
 User=rabbitmq
 Group=rabbitmq
+UMask=0027
```

Comment:
In case it's not obvious to some readers: this makers the files created in the directory non world-readable (or writable, or executable).

Reply:
If log files are owned by `rabbitmq:rabbitmq`, and we disable all world permissions, does it mean that `adm` group will not be able to read these log files?

Reply:
umask only will have effect on newly created files.

Reply:
Thanks for clarifying that @michaelklishin I should have mentioned it. Thanks for the reviews.